README.md

```diff
@@ -8,6 +8,22 @@ The Collective Experience (CE) Platform facilitates the creation and operation o
 4. Run `meteor npm install` to install local dependencies.
 5. Start the server using `meteor`.
 
+Hacky way in case meteor installation doesnt work (for Windows):
+
+apt-get install mongodb-server
```

Comment:
@mtruong68 can you make this nicely markdown formatted? you might use the ` symbol to put things in nice font.
i.e. ```